### PR TITLE
feat: CephFS implementation to set directory quotas

### PR DESCRIPTION
**Pull request by leksikov**
Feature implementation and solution for issue _CephFS quota support_ [#237](https://github.com/lablup/backend.ai/issues/237) and part of [#50](https://github.com/lablup/backend.ai/issues/50).

After the CephFS is installed and vfolder directory is set and mounted. Storage proxy can create folders in designated directory and set the quotas limits.

The guide and script about testing Ceph is at [manual](https://github.com/lablup/backend.ai-storage-proxy/tree/feature/ceph-implementation/scripts/spawn-ceph-clusters)

### CODE REVIEW

**Comment by leksikov** [comment]
Build an installation script for deploying and testing ceph-cluster with storage proxy.

**Comment by leksikov** [comment]
Recent commit provides functionality to create folder on CephFS mounted ceph-fuse volume directory. 
While creating a directory it sets a quota. However, quota functionality cannot be completely tested due to operation not supported issue from setfattr.

**Review by adrysn — CHANGES_REQUESTED** [review]
I have some question on the code for the initial review.

I also think that it would be better to use Ceph REST API to execute commands, but I think I need some time to study about it.

File: src/ai/backend/storage/cephfs/__init__.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            raise RuntimeError("Ceph is not installed. ")
+
+    # ----- volume opeartions -----
+    async def create_vfolder(
+        self, vfid: UUID, options: VFolderCreationOptions = None
+    ) -> None:
+
+        vfpath = self.mangle_vfpath(vfid)
+        await run(f"sudo  mkdir -p {vfpath}")
+        await run(f"sudo  chmod 777 -R {vfpath}")
```
[the remark this answers is not in the document]
Comment (leksikov) [reply src/ai/backend/storage/cephfs/__init__.py:67]: Without setting chmod rights to newly created folder there is an execution error raised ai.backend.storage.exception.ExecutionError: setfattr: /mnt/vfroot/ceph-fuse/b0/72/9c4d2b3f4cfd9aa178d29117f806: Permission denied.

Setting rights for whole ceph-fuse directory doesn't solve the issue while setting rights at the moment of creation fix the problem.

File: src/ai/backend/storage/cephfs/__init__.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        await self.set_quota(vfpath, 100000000)
```
[the remark this answers is not in the document]
Comment (leksikov) [reply src/ai/backend/storage/cephfs/__init__.py]: Made the quota variable for setting quota function

**Pull request closed without merging**